spec/features/dashborad_spec.rb

```diff
@@ -20,6 +20,9 @@
       expect(page).to have_selector('#definitions_list table tbody tr', count: 1)
 
       expect(page).to have_content('There are no working jobs.')
+      expect(page).to have_title('Dashboard « Kuroko 2')
+      expect(page).to have_selector('i.fa.fa-dashboard', text: '')
+      expect(page).to have_content(' Dashboard')
```

Comment:
The ` Dashboard` may be displayed in other places.
How about using `have_selector('h1', text:  /Dashboard/)` ?